cub/cub/device/dispatch/kernels/reduce.cuh

```diff
@@ -464,7 +454,7 @@ CUB_DETAIL_KERNEL_ATTRIBUTES __launch_bounds__(
   int(ChainedPolicyT::ActivePolicy::SingleTilePolicy::BLOCK_THREADS),
   1) void DeterministicDeviceReduceSingleTileKernel(InputIteratorT d_in,
                                                     OutputIteratorT d_out,
-                                                    OffsetT num_items,
+                                                    int num_items,
```

Comment:
Just curious, but why do we set this to `int` and not do the same for the `run_to_run` single tile kernel? Same for the kernel above